core/src/packet/packet_ms.cpp

```diff
@@ -159,6 +159,7 @@ AOPacket *PacketMS::validateIcPacket(AOClient &client) const
     l_args.append(l_incoming_args[5].toString());
     if (client.m_pos != l_incoming_args[5].toString()) {
         client.m_pos = l_incoming_args[5].toString();
+        client.m_pos.replace("../", "");
```

Comment:
Does not account for Windows path escaping using the backslash.
```suggestion
        client.m_pos.replace("../", "").replace("..\\","");
```